### ISSUE TITLE
Broken emoji escaping

### ISSUE THREAD

**Issue by alecgibson** [fb55/entities#561]
We've just tried bumping to v3, but some unicode escaping seems to have broken in the refactor. For example, try escaping this string of unicode characters

> ♒️♓️♈️♉️♊️♋️♌️♍️♎️♏️♐️♑️

v2 of `entites` correctly encoded this as:

```
&#x2652;&#xFE0F;&#x2653;&#xFE0F;&#x2648;&#xFE0F;&#x2649;&#xFE0F;&#x264A;&#xFE0F;&#x264B;&#xFE0F;&#x264C;&#xFE0F;&#x264D;&#xFE0F;&#x264E;&#xFE0F;&#x264F;&#xFE0F;&#x2650;&#xFE0F;&#x2651;&#xFE0F;
```

but v3 seems to strip out all of the "meaningful" characters apart from the first one, just leaving the `&#xFE0F;` for the rest:

```
&#x2652;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;&#xfe0f;
```

**Comment by fb55** [comment]
Thanks for the report! I have a fix in #562, and will publish a new version soon.

**Comment by fb55** [comment]
Published as `3.0.1`
